src/Components/Server/src/ComponentHub.cs

```diff
@@ -259,11 +251,21 @@ private void SetCircuit(CircuitHost circuitHost)
 
         private static Task NotifyClientError(IClientProxy client, string error) => client.SendAsync("JS.Error", error);
 
-        private static class Log
+        private string GetClientErrorMessage(Exception exception, string additionalInformation = null)
```

Comment:
Does this duplicate something we have elsewhere?

As a detail, we don't currently respond with exception details anywhere in this class, because the catch blocks here *never* catch exceptions from user code (LMK if I'm wrong) 😀

Reply:
I checked and we only have a single try..catch. I removed the code supporting this from componenthub

Reply:
removed